pkg/idtools/idtools.go

```diff
@@ -1,11 +1,8 @@
 package idtools // import "github.com/docker/docker/pkg/idtools"
```

Comment:
Can you remove these `// import`  comments, so that we don't have to remove them after the code is moved? (they're mostly ineffective anyway, so not too problematic if missing)